Mathlib/Logic/Equiv/List.lean

```diff
@@ -64,7 +64,7 @@ theorem encode_list_cons (a : α) (l : List α) :
 theorem decode_list_zero : decode (α := List α) 0 = some [] :=
   show decodeList 0 = some [] by rw [decodeList]
 
-@[simp, nolint unusedHavesSuffices] -- Porting note: false positive
+@[simp, nolint unusedHavesSuffices] -- This is a false positive in the unusedHavesSuffices linter.
```

Comment:
I suppose this doesn't need to be a porting note since we'll find it anyway by looking for `nolint` exceptions?